Logging/LogHandler/FileLogHandler.cs

```diff
@@ -10,12 +9,30 @@ namespace Anvil.CSharp.Logging
     /// </summary>
     public class FileLogHandler : AbstractAnvilBase, ILogHandler
     {
+        /// <summary>
+        /// The type of log files the handler should generate.
+        /// </summary>
+        public enum LogType
+        {
+            /// <summary>
+            /// New logs are appended to the existing log file.
+            /// </summary>
+            Append,
+            /// <summary>
+            /// A new log file is created each session, overwriting the previous one if it exists.
+            /// </summary>
+            Replace,
+        }
+
         public const string LOG_CONTEXT_CALLER_DERIVED_TYPE = "{0}";
         public const string LOG_CONTEXT_CALLER_FILE = "{1}";
         public const string LOG_CONTEXT_CALLER_METHOD = "{2}";
         public const string LOG_CONTEXT_CALLER_LINE = "{3}";
 
-        private readonly StreamWriter m_Writer;
+        private StreamWriter m_Writer;
+        private readonly string m_Path;
+        private readonly LogType m_LogType;
+        private readonly bool m_Rotate;
```

Comment:
```suggestion
        private readonly bool m_ShouldRotate;
```
More clear?

Reply:
I was on the fence about this - `shouldRotate`? `useRotation`? `rotateFiles`?

Reply:
my vote is `shouldRotate` but open to alternatives if someone feels strongly